lardataalg/DetectorInfo/DetectorPropertiesStandard.h

```diff
@@ -108,6 +108,11 @@ namespace detinfo {
         Comment("Allows a scaling factor to fudge the drift velocity "
                 "calculation (as suggested by DriftVel Stancari")};
 
+      fhicl::Atom<bool> UseIcarusMicrobooneDriftModel{
+        Name("UseIcarusMicrobooneDriftModel"),
+        Comment("Allows user to decide to use the ICARUS+MicroBooNE drift "
+                "model for velocity calculation as in arXiv:2008.09765")};
```

Comment:
Suggest adding the default value `false` at the parameter declaration here so people who do not use the `detectorproperties.fcl` file below can continue to use their code as is--i.e. preserve backwards compatibility.